radar/web/src/backend/definitions.ts

```
@@ -0,0 +1,204 @@
+export type RadarState = {
+    worldName: string;
+    playerPawns: RadarPlayerPawn[];
+    plantedC4: RadarPlantedC4 | null;
+    c4Entities: RadarC4[];
+    localControllerEntityId: U32 | null;
+};
+export type Usize = number;
+export type S2CMessage =
+    | {
+          type: "response-success";
+          payload: {};
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)